tests/script-based-pre/playback_already_existing/playback_opts.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+# Test that concrete playback -Z concrete-playback executes as expected
+set -o pipefail
+set -o nounset
+
+RS_FILE="modified.rs"
+cp original.rs ${RS_FILE}
+
+echo "[TEST] Generate test..."
+kani ${RS_FILE} -Z concrete-playback --concrete-playback=inplace
+
+echo "[TEST] Only codegen test..."
+kani playback -Z concrete-playback --only-codegen ${RS_FILE} -- kani_concrete_playback
```

Comment:
You probably don't need this for this test

Reply:
5979461e053a02b480dd3d5a81cc6af5ad400611